docs/appendices/release-notes/unreleased.rst

```diff
@@ -112,3 +112,6 @@ Fixes
 - Fixed an issue with client caching which lead to authentication error when
   creating a subscription with bad credentials and ``pg_tunnel`` followed by
   re-creating it second time with the same name and valid credentials.
+
+- Fixed an issue with VARCHAR primary key columns losing length property after
```

Comment:
BIT type is also affected by this issue but I haven't mentioned in changes as it's not reported

Reply:
Please add it anyway - if we know it fixes it we can mention it.

Reply:
added entry and also covered it with a test in https://github.com/crate/crate/pull/12510/commits/8ffc5a31ba7b409023481015e96f4ac065ef3fd7